components/ui-api-layer/main.go

```diff
@@ -75,7 +75,10 @@ func exitOnError(err error, context string) {
 
 func parseFlags(cfg config) {
 	if cfg.Verbose {
-		flag.Set("stderrthreshold", "INFO")
+		err := flag.Set("stderrthreshold", "INFO")
+		if err != nil {
+			glog.Error(errors.Wrap(err, "while parsing flags"))
```

Comment:
Please consider `glog.Fatal`- but it's totally up to you

Reply:
I decided that the error occurring during setting that flag wasn't critical because it wasn't really related to parsing user provided flags and that logging the error without exiting the application would suffice.